notebooks/tel_and_site/subsys_req_ver/tma/SITCOM-1120_az_brake_analysis.ipynb

```diff
@@ -0,0 +1,232 @@
+{
```

Comment:
<p>Can you add a new cell above this one? Add this code to it, please:</p>

<pre class="ql-syntax" spellcheck="false">%load_ext lab_black
%load_ext autoreload
%autoreload 2
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/120/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>No problem, </p>